lib/testmap.py

```diff
@@ -33,6 +33,7 @@
             'fedora-coreos',
             'fedora-33/firefox',
             'rhel-8-4',
+            'rhel-8-5',
```

Comment:
Also please don't leave it hanging in `_manual` when moving to `master`.

Reply:
Fixed